examples/paywall-tester/src/main/java/com/revenuecat/paywallstester/ui/theme/Type.kt

```diff
@@ -2,19 +2,50 @@ package com.revenuecat.paywallstester.ui.theme
 
 import androidx.compose.material3.Typography
 import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.Font
 import androidx.compose.ui.text.font.FontFamily
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.unit.sp
+import com.revenuecat.paywallstester.R
+
+val superMarioFontFamily =
```

Comment:
I think we should just keep this as the standard font for all paywalls

Reply:
🤣 I would totally do this, but we would need to find versions with smaller weights and cursive :P. That's not supported in this font unfortunately